subprojects/logging/src/main/java/org/gradle/internal/logging/util/Log4jBannedVersion.java

```diff
@@ -22,6 +22,6 @@
  */
 public class Log4jBannedVersion {
     public static final String LOG4J2_CORE_COORDINATES = "org.apache.logging.log4j:log4j-core";
-    public static final String LOG4J2_CORE_STRICT_VERSION_RANGE = "[2.15, 3[";
-    public static final String LOG4J2_CORE_PREFERRED_VERSION = "2.15.0";
+    public static final String LOG4J2_CORE_VULNERABLE_VERSION_RANGE = "[2.0, 2.15[";
```

Comment:
Would `[2.0, 2.14.1]` be an easier to read version of this?  Or `[2.0, 2.15)`?

Reply:
Our documentation mentions all these options, I would go with the first one which uses `)` for the exclusive bound.
I am also going to update this following https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45046 to effextively exclude 2.15.0 from the valid versions.
So final would be `[2.0, 2.16)`